packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -267,13 +278,13 @@ public function import_term( $data ) {
 		 * @param array $meta Meta data.
 		 */
 		$data = apply_filters( 'wxr_importer_pre_process_term', $data );
+		$data = $this->topological_sorter->get_mapped_entity( 'term', $data );
```

Comment:
I'm confused about the method names. It's `sorter` but it doesn't sort – just `maps`. Let's come up with a set of terms that meaningfully reflect that entire idea space. Ideally, this line would communicate something to a new person without requiring additional work, such as diving into the source code of that method.

Reply:
Also, could we only source specific values from the sorter instead of blanket-replacing the entire `$data` array? This leads to a section in the code below where we just accept the `parent` value from `$data` instead of explicitly using the mapped parent value. It's an easy way for unprocessed user input to slip through if we miss even a single check in a single code path. We can reduce the likelihood of running into that entire error class by just saying "sorry, `$data['parent']` won't come through, we use `$mapped_parent` instead".